torch/cuda/graphs.py

```diff
@@ -187,7 +187,12 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 
 def make_graphed_callables(
-    callables, sample_args, num_warmup_iters=3, allow_unused_input=False, pool=None
+    callables,
+    sample_args,
+    num_warmup_iters=3,
+    allow_unused_input=False,
+    pool=None,
+    enable_autocast_bwd=False,
```

Comment:
Yea, let's make this None, and only disable on `False`

Reply:
Sounds good